gatsby-theme-landing-page/src/components/divider.js

```diff
@@ -1,3 +1,4 @@
+// TODO currently unused
```

Comment:
I'm not sure we need this right now, but this could be a type of section component that would allow content editors to add spacing/visual separation between parts of a page, so leaving it for now